app/main.py

```diff
@@ -1,6 +1,23 @@
 from typing import Callable
 
-
 def cache(func: Callable) -> Callable:
-    # Write your code here
-    pass
+    storage = {}
+
+    def wrapper(*args):
+        if args in storage:
```

Comment:
Checking if `args` is in `storage` assumes that `args` is hashable. If `args` contains mutable types, this will raise a `TypeError`. Consider converting `args` to a tuple if it isn't already, to ensure it is hashable.